src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -15,9 +17,11 @@ import {getChannelListInfo, getChannelMembers} from '../../../utils/string/Strin
 type ChannelCategory = 'SIGNED' | 'ANONYMOUS';
 
 const useFetchChannelHook = () => {
-  const {localDb, refresh} = useLocalDatabaseHook();
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
   const {getFirstMessage} = useSystemMessage();
+  // const queue = DatabaseQueue.getInstance();
+  const {queue} = useDatabaseQueueHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useDatabaseQueueHook` is a good addition to manage asynchronous tasks in a queue. However, it's important to ensure that the queue is properly managed and doesn't grow indefinitely or block other operations.



<!-- This is an auto-generated comment by OSS CodeRabbit -->